### ISSUE TITLE
Multiple user support

### ISSUE THREAD

**Issue by firestalk** [tchaik/tchaik#47]
Is it able to implement multi-user functionality, so every user can use own playlists, etc. ?

**Comment by dhowden** [comment]
Yes, we currently store playlists in local storage in the browser.   However, there is a backend storage system in dev (for playlists) which allows multiple users can have different playlists of tracks from the same library.
